<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Brain Cancer Detection</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="WidgetImage">
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="WidgetMask">
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="WidgetProbability">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>512</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionOpenFile"/>
   </widget>
   <addaction name="MenuFile"/>
  </widget>
  <action name="ActionOpenFile">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
